docs/src/writing-tests-js.md

```diff
@@ -3,90 +3,147 @@ id: writing-tests
 title: "Writing Tests"
 ---
 
-Playwright assertions are created specifically for the dynamic web. Checks are automatically retried until the necessary conditions are met. Playwright comes with [auto-wait](./actionability.md) built in meaning it waits for elements to be actionable prior to performing actions. Playwright provides a [test](./api/class-test.md) function to declare tests and the [expect](https://jestjs.io/docs/expect) function to write assertions.
+Playwright tests are simple, they
```

Comment:
Make the changes in the other writing-tests* files as well?

Reply:
One step at a time!